src/HotChocolate/Data/src/Data/Filters/Convention/Extensions/FilterConventionDescriptorExtensions.cs

```diff
@@ -82,7 +82,11 @@ public static class FilterConventionDescriptorExtensions
                 .BindRuntimeType<DateTimeOffset?,
                     ComparableOperationFilterInputType<DateTimeOffset?>>()
                 .BindRuntimeType<TimeSpan?, ComparableOperationFilterInputType<TimeSpan?>>();
-
+                .BindRuntimeType<sbyte, ComparableOperationFilterInputType<sbyte>>()
```

Comment:
Can you also add the nullables :) And also, they have a order, first the non nullable then the nullable
```csharp
 .BindRuntimeType<sbyte, ComparableOperationFilterInputType<sbyte>>()
 .BindRuntimeType<ushort, ComparableOperationFilterInputType<ushort>>()
 .BindRuntimeType<uint, ComparableOperationFilterInputType<uint>>()
 .BindRuntimeType<ulong, ComparableOperationFilterInputType<ulong>>()
....
 .BindRuntimeType<TimeSpan?, ComparableOperationFilterInputType<TimeSpan?>>();
 .BindRuntimeType<sbyte?, ComparableOperationFilterInputType<sbyte?>>()
 .BindRuntimeType<ushort?, ComparableOperationFilterInputType<ushort?>>()
 .BindRuntimeType<uint?, ComparableOperationFilterInputType<uint?>>()
 .BindRuntimeType<ulong?, ComparableOperationFilterInputType<ulong?>>()
```

Reply:
Sure :-)
I also refactored the code a tiny bit to make it more obvious that all comparable types have also their nullable counterpart added. (But I can also revert that back if you don't like it)